vpd-manager/src/ipz_parser.cpp

```diff
@@ -272,6 +272,8 @@ std::pair<types::RecordOffsetList, types::InvalidRecordList>
             // Verify the ECC for this Record
             if (!recordEccCheck(itrToPT))
             {
+                logging::logMessage(
```

Comment:
We already maintain a list of invalid records, can that be used here instead of logging one at a time.